### ISSUE TITLE
508 Defect 2, screen readers, when errors occur, focus isn't placed consistently on the error message

### ISSUE THREAD

**Issue by AngelaFowler82** [department-of-veterans-affairs/va.gov-team#26809]
Label: 15247

    - [508-defect-2]

https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-2

Issue Title: 508 Defect 2, screen readers, when errors occur, focus isn't placed consistently on the error message.
Point of Contact: Angela Fowler
**VFS Point of Contact:** _First name only: Angela

<h2>User Story or Problem Statement:</h2>
As a screen reader user, when encountering an error message which JAWS indicates has a Close button,  my first inclination would be to close out the message. With focus placed on the message I can find the close button easily. When focus isn't managed, it defaults to the top of the page, and it takes several seconds of searching to locate the close button.

<h2>Details</h2>
On Profile 2.0, when a screen reader user attempts to change their address, phone number or email, they may be presented with an error message effectively stating we are unable to make the requested change. This message comes with a close button. The majority of the time, focus is placed at the top of the page, causing the user to search through several buttons before they come to "close."

<h2>Acceptance Criteria</h2>
In the event of an error, focus is consistently placed on the error message.

<h2>Environment</h2>
Consistent across all environments.

<h2>Steps to Recreate</h2>
<ul>
<li>Log into Profile 2.0 using vets.gov.user+1@gmail.com</li>
<li> Using JAWS or NVDA, Attempt to add an address, phone number, or email address.</li>
</ul>

**Comment by mtcA6** [comment]
Notes on pointing: feels like a 3, making sure the focus goes to the right place, seems like a small change in one spot but there's a little bit of investigation to locate, and working w/the specific screen reader called out might be a challenge if it doesn't work w/mac

**Comment by ACParker89** [comment]
We will need a screen reader user with JAWS in order to complete this ticket.

@ajakabcin - have you all used JAWS before/know anyone with it?

**Comment by ACParker89** [comment]
https://dsva.slack.com/archives/C909ZG2BB/p1726492137038769
